.github/workflows/linting.yml

```diff
@@ -8,85 +8,8 @@ on:  # yamllint disable-line rule:truthy
 
 jobs:
 
-  flake8:
```

Comment:
Any reason to remove them here? As fas as I know, installation of `pre-commit` is not required, right?

Reply:
they are just duplicates to what the pre-commit does, so you want to jeep both?
note that your pre-cimmot[bot] is called on each PR too

Reply:
![image](https://user-images.githubusercontent.com/6035284/158974423-45d4c4ea-f809-4add-8200-a8aaa89d21c5.png)

Reply:
Yes, thanks for clarifying. It is indeed good to remove them.

Reply:
The required checks should be resolved.